src/inmanta/server/services/environmentservice.py

```diff
@@ -268,6 +289,7 @@ def get_depended_by(self) -> List[str]:
             metadata = model.ModelMetadata(message="Decommission of environment", type="api")
         version_info = model.ModelVersionInfo(export_metadata=metadata)
         await self.orchestration_service.put_version(env, version, [], {}, [], version_info.dict(), get_compiler_version())
+        await self.notify_listeners(EnvironmentAction.updated, env.to_dto())
```

Comment:
Do we need a notification for a decommission operation? I am not sure. @wouterdb wdyt?

Reply:
Not sure either.

Reply:
So perhaps leave it out to keep it simple?